### ISSUE TITLE
Should we accommodate 3:2 screen aspect ratio?

### ISSUE THREAD

**Issue by joelostblom** [UBC-MDS/exploratory-data-viz#117]
Some new computers, particularly Surfaces and Macs, have a 3:2 aspect ratio. This compresses the slides further so that some plots or labels end up outside the screen. We currently have tested on 16:9 and 16:10. I can test on 3:2 also if we think it is worthwhile, I'll bring this up on friday.

**Comment by joelostblom** [comment]
How was this handled in previous course @hfboyce ? It probably occurs less often but I imagine that there are at least some slides with full screen tables or plots where this might have been an issue?

**Comment by hfboyce** [comment]
I don’t think this did ever happen to my knowledge? We rarely had full page tables and the plots were displayed as images where I would decrease the image size.

Basically to answer you question is we didn’t ever encounter this o my knowledge so we never had a course of action.

**Comment by joelostblom** [comment]
The slides that look worse in 3:2 is when the narrower aspect ratios break title lines into two. One solution to make the variability issues go away is to fix the max-width of the slides to what looks good on 3:2 and then shorten all the titles. We would probably also need to adjust the max height to what looks good on 16:9.

 This should look good on all screen, but it gives us the least slide space overall and it is going to challenging to shorten some of the titles enough. For recording, we should keep whatever looks good in the recording 16:9 window, so I will look into this after recording is done.

Currently, we have optimized for 16:10 with a persistent dock and top bar on a Mac, which means that Win computers would probably be fine since they have more vertical space available by default.
